src/main.zig

```diff
@@ -7,11 +8,13 @@ const Translator = @import("Translator.zig");
 
 const fast_exit = @import("builtin").mode != .Debug;
 
-var general_purpose_allocator: std.heap.GeneralPurposeAllocator(.{}) = .init;
+var debug_allocator: std.heap.DebugAllocator(.{}) = .init;
 
 pub fn main() u8 {
-    const gpa = general_purpose_allocator.allocator();
-    defer _ = general_purpose_allocator.deinit();
```

Comment:
We should still deinit `debug_allocator` in Debug mode.

Reply:
Woooops indeed